src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -20,9 +20,39 @@ public LineController(final LineService lineService) {
         this.lineService = lineService;
     }
 
+
     @PostMapping
-    public ResponseEntity createLine(@RequestBody LineRequest lineRequest) {
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
         LineResponse line = lineService.saveLine(lineRequest);
         return ResponseEntity.created(URI.create("/lines/" + line.getId())).body(line);
     }
+
+    @GetMapping(produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<LineResponse>> showLines() {
+        return ResponseEntity.ok().body(lineService.findAllLines());
+    }
+
+    @DeleteMapping(value = "/{id}")
+    public ResponseEntity deleteStation(@PathVariable Long id) {
```

Comment:
```suggestion
    public ResponseEntity deleteLine(@PathVariable Long id) {
```

method 명이 혼선을 줄 수 있을 거 같아요!